### PR TITLE
Add ces context docs

### LINKED ISSUE #3: Add documentation for CES context

### PR DESCRIPTION
**Pull request by nhinze23**
This commit adds the ces context docs. Most content from the docs is already documented.
The basic chapter links now to the introduction of docs.cloudogu.com, hints the location of logfiles from the dogus and gives a outlook of a rich integrated dogu.

Resolves #3

### CODE REVIEW

File: docs/core/basics_de.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Einleitung & Basics
 
-Ein Dogu (aus dem Japanischen abgeleitet für „Werkzeug“) stellt eine containerisierte Anwendung im Cloudogu EcoSystem (CES) zur Verfügung.
+Ein Dogu (aus dem Japanischen abgeleitet für „Werkzeug“) stellt eine containerisierte Anwendung im [Cloudogu EcoSystem (CES)](https://docs.cloudogu.com/de/docs/introduction/) zur Verfügung.
```
Comment (ppxl) [inline docs/core/basics_de.md:3]: Nice!

File: docs/core/basics_de.md [hunk 2]
```diff
@@ -204,6 +204,19 @@ python3 -m http.server 8080 --directory /resources/web
...
\ No newline at end of file
+
+Die Syslog-Konfiguration des Cloudogu EcoSystems sorgt dafür, dass jedes Dogu ein einzelnes Logfile unter dem Pfad
+`/var/log/docker/` schreibt, um bspw. Log-Forwarding oder Analysen zu ermöglichen.
+
+In diesem Log können nun Ausgaben geprüft werden, ob das Dogu korrekt gestartet wurde:
+- `cat /var/log/docker/newdogu.log`
+
+Alternativ kann auch der HTTP-Endpunkt des Dogus aufgerufen werden:
+- `https://192.168.56.2/newdogu` aufrufen.
```
Comment (ppxl) [inline docs/core/basics_de.md:215]: Checking the output of `docker ps -a` would also make a good start.

File: docs/core/basics_de.md [hunk 3]
```diff
@@ -204,6 +204,19 @@ python3 -m http.server 8080 --directory /resources/web
...
+
+## Architektursicht eines integrierten Dogus
+
+Das folgende Schaubild zeigt mögliche Kommunikationswege eines integrierten Dogus (Redmine) im Cloudogu EcoSystem.
+Im Gegensatz zu dem minimalen Dogu aus dem vorherigen Kapitel besitzt Redmine andere Dogus als Abhängigkeiten, verwendet Service-Accounts und Benutzerinformationen über das CAS-Dogu und wird über den ETCD konfiguriert.
```
Comment (ppxl) [inline docs/core/basics_de.md:220]: I'd like to emphasize that now more than ever, the existance of ETCD as local registry is not as fixed as we used to think. I suggest to replace ETCD with a more abstract term.

```suggestion
Im Gegensatz zu dem minimalen Dogu aus dem vorherigen Kapitel besitzt Redmine andere Dogus als Abhängigkeiten, verwendet Service-Accounts und Benutzerinformationen über das CAS-Dogu und wird über die EcoSystem-Registry konfiguriert.
```

**Review by ppxl — CHANGES_REQUESTED** [review]
Hiya! Thanks for your work. There wasn't much left for the original "context chapter" :rofl: 

Anyhow, there is still this link in the readme to a non-existent context document that needs to be removed: https://github.com/cloudogu/dogu-development-docs/blob/feature/3-add-ces-context-docs/README.md?plain=1#L11

But then we have finally made it! :tada:

File: docs/core/basics_de.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Einleitung & Basics
 
-Ein Dogu (aus dem Japanischen abgeleitet für „Werkzeug“) stellt eine containerisierte Anwendung im Cloudogu EcoSystem (CES) zur Verfügung.
+Ein Dogu (aus dem Japanischen abgeleitet für „Werkzeug“) stellt eine containerisierte Anwendung im [Cloudogu EcoSystem (CES)](https://docs.cloudogu.com/de/docs/introduction/) zur Verfügung.
 Dogus können unterschiedliche Nutzungsaspekte befriedigen, wie z.B. eine Projektverwaltung (gegenüber Benutzenden) oder ein Datenbankmanagementsystem (gegenüber anderen Dogus).
 Daraus schließt sich, dass Dogus mehr können müssen als mit reinen Docker-Direktiven möglich wäre. Denkt man an die obigen Dogus, ist es unter anderem sinnvoll einen ServiceAccount-Mechanismus für die Kommunikation von
```
Comment (alexander-dammeier) [inline docs/core/basics_de.md:5]:
```suggestion
Daraus schließt sich, dass Dogus mehr können müssen, als mit reinen Docker-Direktiven möglich wäre. Denkt man an die obigen Dogus, ist es unter anderem sinnvoll einen ServiceAccount-Mechanismus für die Kommunikation von 
```

File: docs/core/basics_en.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Introduction & Basics
 
-A dogu (derived from Japanese for "tool") provides a containerized application in the Cloudogu EcoSystem (CES).
+A dogu (derived from Japanese for "tool") provides a containerized application in the [Cloudogu EcoSystem (CES)](https://docs.cloudogu.com/en/docs/introduction/).
 Dogus can satisfy different usage aspects, such as a project management (towards users) or a database management system (towards other dogus).
 It follows that dogus must be able to do more than would be possible with pure Docker directives. Thinking of the above dogus, it makes sense, among other things, to have a ServiceAccount mechanism for communication from
```
Comment (alexander-dammeier) [inline docs/core/basics_en.md:5]: Sounds not quite right to me without the additional `what` but i may be wrong here.
```suggestion
It follows that dogus must be able to do more than what would be possible with pure Docker directives. Thinking of the above dogus, it makes sense, among other things, to have a ServiceAccount mechanism for communication from
```

**Review by alexander-dammeier — CHANGES_REQUESTED** [review]
Nice work :)
Just 2 little enhancements and then i can merge it.

File: docs/core/basics_en.md (hunk 5 above)
Comment (nhinze23) [reply docs/core/basics_en.md:5]: Mhh.. For me it sounds wrong with the `what`..
Let us trust deepl :smile:

**Review by alexander-dammeier — APPROVED** [review]

**Pull request merged**